<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Libkface GUI Test Program</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="openImageBtn">
          <property name="text">
           <string>Open Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="detectFacesBtn">
          <property name="text">
           <string>Detect Faces</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recogniseBtn">
          <property name="text">
           <string>Recognise</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updateDatabaseBtn">
          <property name="text">
           <string>Update Database</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveConfigBtn">
          <property name="text">
           <string>Save Config</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="configLocation">
        <property name="text">
         <string>Some Default File Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="openConfigBtn">
        <property name="text">
         <string>Config Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
